app/components/shared/inputs/ColorInput.tsx

```diff
@@ -1,47 +1,93 @@
+import cx from 'classnames';
 import { Component, Prop } from 'vue-property-decorator';
 import { Sketch } from 'vue-color';
 import { BaseInput } from './BaseInput';
-import { IInputMetadata } from './index';
+import { IColorMetadata } from './index';
+import styles from './ColorInput.m.less';
+import { $t } from 'services/i18n';
+import { Inject } from 'services';
+import { UsageStatisticsService } from 'services/usage-statistics';
+import { loadColorPicker } from 'util/slow-imports';
+import { OS, getOS } from 'util/operating-systems';
 
 @Component({})
-export default class ColorInput extends BaseInput<string, IInputMetadata> {
+export default class ColorInput extends BaseInput<string, IColorMetadata> {
   @Prop() readonly value: string;
-  @Prop() readonly metadata: IInputMetadata;
+  @Prop() readonly metadata: IColorMetadata;
   @Prop() readonly title: string;
+  @Inject() readonly usageStatisticsService: UsageStatisticsService;
 
   pickerVisible = false;
 
   togglePicker() {
     this.pickerVisible = !this.pickerVisible;
   }
 
-  get swatchStyle() {
-    return {
-      backgroundColor: this.value,
-    };
+  async eyedrop(e: MouseEvent) {
+    e.stopPropagation();
+    const colorPicker = (await loadColorPicker()).default;
+    this.usageStatisticsService.recordFeatureUsage('screen-color-picker');
```

Comment:
Minor thing, but we're using CamelCase for feature names.  Would be good for this to match.